docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,166 @@
   not have the full context. Legible code is more reusable, bug-free,
   and future-proof.
 
 - **Accidental errors** (e.g., typos) as well as **structural errors**
-  (e.g., dead code, logic or algorithm bugs) are often much easier to
-  spot for critical reviewers with an outside perspective. (Often,
-  it's a rather satisfying sport to find bugs...)
+  (e.g., dead code, logic or algorithm bugs, performance or architecture
+  concerns) are often much easier to
+  spot for critical reviewers with an outside perspective.
+  Studies have found that even short and informal code reviews have significant
+  [impact on code quality and bug frequency](https://blog.codinghorror.com/code-reviews-just-do-it/).
+
+- **Security reviews** are required in some environments. They are performed
+  by application security engineers following industry-standard
+  procedures and frameworks.
+
 
-## What to Review
+## What to review
 
 There is no eternally true answer to this question and the right
-approach should be agreed on by each development team; some teams prefer
-to review *every* CR merged into the develop branch, while others will have a
+approach should be agreed on by each development team. Some teams prefer
+to review every change merged into the main branch, while others will have a
 "triviality" threshold under which a review is not required. The
 trade-off is between effective use of engineers' (both authors' and
-reviewers') time and maintaining code quality.
+reviewers') time and maintaining code quality. In certain regulatory environments,
+code review may be required even for trivial changes.
+
+Code reviews are classless: being the most senior person on the team does not
+imply that your code does not need review. Even if, in the rare case, code is
+flawless, the review provides an opportunity for mentorship and collaboration,
+and, minimally, diversifies the understanding of code in the code base. 
+
 
-## Preparing Code for Review
+## When to review
 
-It is the author's duty to submit CRs that are easy to review in order
-not to waste the reviewers' time and motivation. A few guidelines:
+Code reviews should happen after automated checks (tests, style, other CI)
+have completed successfully, but before the code merges to the repository's
+mainline branch. We generally don't perform formal code review of aggregate
+changes since the last release.
 
-- **Scope and size:** Changes have a narrow, well-defined, self-contained
-  scope that they cover exhaustively. For example, a CR may implement
-  a new feature or fix a bug. Shorter CRs are preferred over
-  longer CRs. If a CR makes substantive changes to more than ~5
+For complex changes that should merge into the mainline branch as a single unit
+but are too large to fit into one reasonable CR, consider a stacked CR model:
+Create a primary branch `feature/big-feature` and a number of secondary branches
+-- e.g., `feature/big-feature-api`, `feature/big-feature-testing`, etc. -- that
```

Comment:
nit: use em dashes to be fancier, and if you do, don't include spaces around them

```
Create a primary branch `feature/big-feature` and a number of secondary branches&mdash;e.g.,
`feature/big-feature-api`, `feature/big-feature-testing`, etc.&mdash;that each
```

Reply:
replaced by `()`